# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../evas/disable-GL_NV_texture-use.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


w/ XFree86 Radeon hardware acceleration (at least on iBook) the
NV extension does produce major pixel garbage.

  - Rene Rebe <rene@exactcode.de>

--- evas-2004-02-25/src/lib/engines/gl_common/evas_gl_context.c.orig	2004-02-27 03:12:11.000000000 +0100
+++ evas-2004-02-25/src/lib/engines/gl_common/evas_gl_context.c	2004-02-27 03:12:49.000000000 +0100
@@ -64,7 +64,7 @@
 	if (ext)
 	  {
 	     if (strstr(ext, "GL_SGIS_generate_mipmap")) gc->ext.sgis_generate_mipmap = 1;
-	     if (strstr(ext, "GL_NV_texture_rectangle")) gc->ext.nv_texture_rectangle = 1;
+	     if (strstr(ext, "GL_NV_texture_rectangle")) gc->ext.nv_texture_rectangle = 0;
 	     /* technically this should work, as its a compatible */
 	     /* implementation of the nvidia texture_rectangle extension */
 	     /* since the #define value is the same as is the description */
